src/Header.js

```diff
@@ -0,0 +1,15 @@
+const Header = () => {
+  return (
+    <header className="Header">
+      <div className="wrapper">
+        <h2>Syde Kicks</h2>
+        <p>Lorem ipsum dolor sit amet.</p>
+        <a href="#collection" className="ctaBtn">
```

Comment:
This code is really GREAT! I can use this as a study material. Maybe instead of using <a href>, use a <Link/>. But if there is reason, <a href> is better, please let me know. I can learn from it as well.